site/docs/reference/Connectors/materialization-connectors/google-cloud-sql-sqlserver.md

```diff
@@ -0,0 +1,209 @@
+# Google Cloud SQL for SQLServer
+
+This connector materializes Flow collections into tables in a Microsoft SQLServer database.
```

Comment:
probably "Google Cloud SQL for SQLServer database"

Reply:
I'm not going to incorporate this since this is what Google Cloud calls it